awxkit/awxkit/api/pages/organizations.py

```diff
@@ -39,10 +39,20 @@ def remove_galaxy_credential(self, credential):
                 "disassociate": True,
             })
 
-
     def payload(self, **kwargs):
         payload = PseudoNamespace(name=kwargs.get('name') or 'Organization - {}'.format(random_title()),
                                   description=kwargs.get('description') or random_title(10))
+        
+        for fk_field in ('default_environment'):
```

Comment:
I found a bug, I need to fix it before merge. Basically when I have only one parameter, it cannot use the variable. @kdelee

Reply:
You need a comma for it to be a tuple.  `for fk_field in ('default_environment',):`